---
title: "Imputation-Based Analysis of Association Studies: Candidate Regions and Quantitative Traits"
date: 2007-07-01
publishDate: 2020-06-04T14:15:26.583985Z
authors: ["Bertrand Servin", "Matthew Stephens"]
publication_types: ["2"]
abstract: "We introduce a new framework for the analysis of association studies, designed to allow untyped variants to be more effectively and directly tested for association with a phenotype. The idea is to combine knowledge on patterns of correlation among SNPs (e.g., from the International HapMap project or resequencing data in a candidate region of interest) with genotype data at tag SNPs collected on a phenotyped study sample, to estimate (“impute”) unmeasured genotypes, and then assess association between the phenotype and these estimated genotypes. Compared with standard single-SNP tests, this approach results in increased power to detect association, even in cases in which the causal variant is typed, with the greatest gain occurring when multiple causal variants are present. It also provides more interpretable explanations for observed associations, including assessing, for each SNP, the strength of the evidence that it (rather than another correlated SNP) is causal. Although we focus on association studies with quantitative phenotype and a relatively restricted region (e.g., a candidate gene), the framework is applicable and computationally practical for whole genome association studies. Methods described here are implemented in a software package, Bim-Bam, available from the Stephens Lab website http://stephenslab.uchicago.edu/software.html."
featured: true
publication: "*PLOS Genetics*"
tags: ["haplotypes", "alleles", "genome-wide association studies", "phenotypes", "dna recombination", "molecular genetics", "permutation", "variant genotypes"]
url_pdf: "https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0030114"
doi: "10.1371/journal.pgen.0030114"
---

