---
title: "A genome-wide association study of production traits in a commercial population of Large White pigs: evidence of haplotypes affecting meat quality."
date: 2014-01-01
publishDate: 2020-06-04T14:15:26.563018Z
authors: ["Marie-Pierre Sanchez", "Thierry Tribout", "Nathalie Iannuccelli", "Marcel Bouffaud", "Bertrand Servin", "Amabel Tenghe", "Patrice Dehais", "Nelly Muller", "Maria Pilar Del Schneider", "Marie-José Mercat", "Claire Rogel-Gaillard", "Denis Milan", "Jean-Pierre Bidanel", "Hélène Gilbert"]
publication_types: ["2"]
abstract: "BACKGROUND: Numerous quantitative trait loci (QTL) have
been detected in pigs over the past 20 years using microsatellite
markers. However, due to the low density of these markers, the
accuracy of QTL location has generally been poor. Since 2009, the
dense genome coverage provided by the Illumina PorcineSNP60 BeadChip
has made it possible to more accurately map QTL using genome-wide
association studies (GWAS). Our objective was to perform high-density
GWAS in order to identify genomic regions and corresponding haplotypes
associated with production traits in a French Large White population
of pigs. METHODS: Animals (385 Large White pigs from 106 sires) were
genotyped using the PorcineSNP60 BeadChip and evaluated for 19 traits
related to feed intake, growth, carcass composition and meat
quality. Of the 64,432 SNPs on the chip, 44,412 were used for GWAS
with an animal mixed model that included a regression coefficient for
the tested SNPs and a genomic kinship matrix. SNP haplotype effects in
QTL regions were then tested for association with phenotypes following
phase reconstruction based on the Sscrofa10.2 pig genome
assembly. RESULTS: Twenty-three QTL regions were identified on
autosomes and their effects ranged from 0.25 to 0.75 phenotypic
standard deviation units for feed intake and feed efficiency (four
QTL), carcass (12 QTL) and meat quality traits (seven QTL). The 10
most significant QTL regions had effects on carcass (chromosomes 7,
10, 16, 17 and 18) and meat quality traits (two regions on chromosome
1 and one region on chromosomes 8, 9 and 13). Thirteen of the 23 QTL
regions had not been previously described. A haplotype block of 183 kb
on chromosome 1 (six SNPs) was identified and displayed three distinct
haplotypes with significant (0.0001 < P < 0.03) associations with all evaluated meat quality traits. CONCLUSIONS: GWAS analyses with the PorcineSNP60 BeadChip enabled the detection of 23 QTL regions that affect feed consumption, carcass and meat quality traits in a LW population, of which 13 were novel QTL. The proportionally larger number of QTL found for meat quality traits suggests a specific opportunity for improving these traits in the pig by genomic selection."
featured: false
publication: "*Genetics, selection, evolution : GSE*"
url_pdf: "http://www.gsejournal.org/content/46/1/12"
doi: "10.1186/1297-9686-46-12"
---

